syntax/grammar.mjs

```diff
@@ -64,7 +66,10 @@ let Term = defer(() =>
 let Comment = defer(() =>
     repeat1(
         sequence(
-            string("#"),
+            either(
+                string("#"),
+                string("//"), // pre 0.5
```

Comment:
It would be nice to have this comment in the generated EBNF, too. Maybe even add a new production dedicated to the legacy `//` comment? Although this would likely complicate the definition of `Message` and `Term`, so maybe it's not worth it.

Reply:
Argh, re-reading, I realize it might not be clear what `comment` refers to above. I'll try to rephrase:

> It would be nice to have this comment (i.e. `// pre 0.5`) in the generated EBNF, too. Maybe even add a new production dedicated to the legacy pre-0.5 Comment?

Reply:
Wasn't too hard, and I did a complete rebase-history thing, as I think the individual commits are good to take in the end. And reviewing the follow-up commit would have been hard, I guess.